app/Makefile

```diff
@@ -228,10 +228,15 @@ help: ## Prints the help documentation and info about each command
 # ETL
 ##################################################
 
+check-dataset-id:
+ifndef DATASET_ID
+	$(error DATASET_ID is undefined)
+endif
+
```

Comment:
Suggesting changing this to `program` for consistency

Also threading arbitrarily, we should also update the example in the root README.md to include this parameter.